### PR TITLE
Fix the inconsistent scaling_rule.adjustment setting in API server and Scaling Engine

### PR DESCRIPTION
**Pull request by cdlliuy**
#156429831

### CODE REVIEW

**Comment by paltanmoy** [comment]
@cdlliuy Lets discuss this in detail before we go ahead.

File: api/lib/validation/schemaValidator.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ var getValidOperators = function() {
 };
 
 var getAdjustmentPattern = function() {
-  var adjustmentPattern = '^[-+][1-9]+[0-9]*$';
+  var adjustmentPattern = '^[-+][1-9]+[0-9]*[%]*$';
```
Comment (boyang9527) [inline api/lib/validation/schemaValidator.js]: this regular expression allows multiple %

**Comment by boyang9527** [comment]
LGTM

**Comment by qibobo** [comment]
LGTM

**Pull request merged**